### PR TITLE
feat: CLI add doc blocks

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1748

### Description:
CLI add doc blocks
### Preview:

Peek 2022-05-27 09-44

### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD  
- [x] CLI

### CODE REVIEW

**Comment by rephus** [comment]
@owlas I'm still in the process to generate the doc on the description. 2 questions: 

- How are we making this optional ? `users are able to turn off automatically adding doc blocks.` 
- what should be the logic for merging `newColumn` with `existingColumns` ?

**Comment by rephus** [comment]
Implemented doc overwrite confirmation as suggested here

`So, they told you all the models + columns they found doc blocks for, then they asked you if you wanted them added in.`

**Comment by owlas** [comment]
@rephus nice warning message! We could add the option to skip all prompts with something like:

* `-y, --yes answers yes to all prompts`
* `-f, --force overwrites existing files without prompting`

File: packages/cli/src/dbt/models.ts [hunk 1]
```diff
@@ -50,23 +52,67 @@ export const getWarehouseTableForModel = async ({
...
+};
+
+const askOverwriteDescription = async (
+    columnName: string,
+    existingDescription: string | undefined,
+    newDescription: string | undefined,
+    spinner: ora.Ora,
+): Promise<string> => {
+    if (!existingDescription) return newDescription || '';
+    if (!newDescription) return existingDescription;
```
Comment (owlas) [inline packages/cli/src/dbt/models.ts:88]: @rephus what if the existing and new match?

**Comment by owlas** [comment]
Works really well!

**Pull request merged**